modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/loader/XmlExtensionLoaderDelegate.java

```diff
@@ -71,14 +72,23 @@
   protected static final String CONFIG_NAME = "config";
 
   private static final ClassTypeLoader typeLoader = ExtensionsTypeLoaderFactory.getDefault().createTypeLoader();
-  private static final Map<String, MetadataType> types = ImmutableMap.<String, MetadataType>builder()
+  private static final Map<String, MetadataType> defaultInputTypes = ImmutableMap.<String, MetadataType>builder()
```

Comment:
Create a variable with the common set of types and the reuse it for defaultInputTypes and defaultOutputTypes